### DISCUSSION TITLE
What are these configuration errors and how do I overcome them?

### DISCUSSION THREAD

**Discussion by mobynet1** [unoplatform/uno#18086] [General]
I just took a look at my log file and see a raft of errors such as:

```text
ReadPackageFileAsync - File 'appsettings.keyvaluestorageconfiguration.json' does not exist in package
ReadPackageFileAsync - File 'appsettings.appconfig.json' does not exist in package
ReadPackageFileAsync - File 'appsettings.localizationconfiguration.json' does not exist in package
ReadPackageFileAsync - File 'appsettings.localizationsettings.json' does not exist in package
ReadPackageFileAsync - File 'appsettings.navigationconfiguration.json' does not exist in package
```

It would seem there is some configuration that I could or should be doing, but I can't find any documentation on how to handle these errors. They seem harmless, but I abhor errors in my log file that are not of my own making :-).

Thanks in advance for your assistance!
Steve

**Comment by jeromelaban** [comment]
Thanks for the report.

Those are information messages that do not need fixing, an upcoming version of Uno will remove those messages.

**Comment by mobynet1** [comment]
@jeromelaban First off, I need to tell you that I am so impressed with the whole Uno Platform team! You are so responsive! I have bugs that I opened over two years ago with the Maui team that they still don't have fixed, I ask them questions that they do not respond to... it was the driving factor into my abandoning the Maui codebase altogether and going to Uno. It is a breath of fresh air!

Also, thank you for your response. I figured this was the case the messages being logged were harmless. Thanks for confirming that!

Cheers,
Steve

**Reply by jeromelaban** [reply]
Thanks for your kind words!
